pages/agent/v3/help/_bootstrap.md

```diff
@@ -60,11 +60,12 @@ See https://buildkite.com/docs/agent/v3/hooks for more details.
 <tr id="clean-checkout"><th><code>--clean-checkout </code> <a class="Docs__attribute__link" href="#clean-checkout">#</a></th><td><p>Whether or not the bootstrap should remove the existing repository before running the command<br /><strong>Environment variable</strong>: <code>$BUILDKITE_CLEAN_CHECKOUT</code></p></td></tr>
+<tr id="git-checkout-flags"><th><code>--git-checkout-flags value</code> <a class="Docs__attribute__link" href="#git-checkout-flags">#</a></th><td><p>Flags to pass to "git checkout" command (default: "-f")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_CHECKOUT_FLAGS</code></p></td></tr>
 <tr id="git-clone-flags"><th><code>--git-clone-flags value</code> <a class="Docs__attribute__link" href="#git-clone-flags">#</a></th><td><p>Flags to pass to "git clone" command (default: "-v")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_CLONE_FLAGS</code></p></td></tr>
 <tr id="git-clone-mirror-flags"><th><code>--git-clone-mirror-flags value</code> <a class="Docs__attribute__link" href="#git-clone-mirror-flags">#</a></th><td><p>Flags to pass to "git clone" command when mirroring (default: "-v")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_CLONE_MIRROR_FLAGS</code></p></td></tr>
 <tr id="git-clean-flags"><th><code>--git-clean-flags value</code> <a class="Docs__attribute__link" href="#git-clean-flags">#</a></th><td><p>Flags to pass to "git clean" command (default: "-ffxdq")<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_CLEAN_FLAGS</code></p></td></tr>
 <tr id="git-fetch-flags"><th><code>--git-fetch-flags value</code> <a class="Docs__attribute__link" href="#git-fetch-flags">#</a></th><td><p>Flags to pass to "git fetch" command<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_FETCH_FLAGS</code></p></td></tr>
-<tr id="git-submodule-clone-config"><th><code>--git-submodule-clone-config value</code> <a class="Docs__attribute__link" href="#git-submodule-clone-config">#</a></th><td><p>Comma separated key=value git config pairs applied before git submodule clone commands, e.g. `update --init`. If the config is needed to be applied to all git commands, supply it in a global git config file for the system that the agent runs in instead.<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_SUBMODULE_CLONE_CONFIG</code></p></td></tr>
+<tr id="git-submodule-clone-config"><th><code>--git-submodule-clone-config value</code> <a class="Docs__attribute__link" href="#git-submodule-clone-config">#</a></th><td><p>Comma separated key=value git config pairs applied before git submodule clone commands, e.g. <code>update --init</code>. If the config is needed to be applied to all git commands, supply it in a global git config file for the system that the agent runs in instead.<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_SUBMODULE_CLONE_CONFIG</code></p></td></tr>
```

Comment:
Nitpick while you're here: It's best to avoid "e.g." and "i.e." because people confuse them.

```suggestion
<tr id="git-submodule-clone-config"><th><code>--git-submodule-clone-config value</code> <a class="Docs__attribute__link" href="#git-submodule-clone-config">#</a></th><td><p>Comma separated key=value git config pairs applied before git submodule clone commands. For example, <code>update --init</code>. If the config is needed to be applied to all git commands, supply it in a global git config file for the system that the agent runs in instead.<br /><strong>Environment variable</strong>: <code>$BUILDKITE_GIT_SUBMODULE_CLONE_CONFIG</code></p></td></tr>
```